components/switch/switch-mixin.js

```diff
@@ -40,6 +40,7 @@ export const SwitchMixin = superclass => class extends FocusMixin(RtlMixin(Focus
 			:host {
 				display: inline-block;
 				white-space: nowrap;
+				user-select: none;
```

Comment:
I could also separately just add this to `.d2l-switch-container` and `.d2l-switch-text`, if that's preferred.